conf/modules/assembly_variant_calling.config

```diff
@@ -13,6 +13,10 @@
 process {
 
     withName: DIPCALL {
+
+        // We need to remove this when running the test profile
+        ext.args = '--cs'
+
         publishDir = [
             path: { "${params.outdir}/assembly_variant_calling/dipcall/${meta.id}" },
             mode: params.publish_dir_mode,
```

Comment:
I think its ok if you don't take away the comment as long as you also have this module in your test.config